helm_deploy/help-with-prison-visits-asynchronous-worker/templates/daily-tasks-cron.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "app.fullname" . }}-cron-daily-tasks
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+spec:
+  schedule: "30 4 * * *"
```

Comment:
Does this run at weekends also?

Reply:
Apparently so

Reply:
is there a conflict between the job history limits and the ttlSecondsAfterFinished?  Won't this delete each job 5 mins after completion, so eventually you wont have much of a job history?  Especially considering this runs at 04:30 in the morning, by the time someone comes in to look at the logs, the job will have been cleaned up?

Reply:
Interesting, never thought of that :D